### DISCUSSION TITLE
DeDRM_plugin.zip who?

### DISCUSSION THREAD

**Discussion by asinbea** [apprenticeharper/DeDRM_tools#1475] [Q&A]
Hello,
might ask the stupidest question but I really can't find the .zip of the DeDRM_plugin to load it into calibre. In the folder there is only "alfcrypto_src" as a .zip. 

Best,
Andre

**Comment by ElleKayEm** [comment]
If you are trying to load the 7.0.3 beta for calibre 5.x, don't unzip it.  It is the DeDRM plugin only.  Load the whole zipped up file.

**Reply by asinbea** [reply]
I already tried this but calibre says: 
ERROR: Unhandled exception: <b>InvalidPlugin</b>:The plugin in '/cd/DeDRM_tools_6.7.0 (1).zip' is invalid. It does not contain a top-level __init__.py file

**Reply by christinaanne58** [reply]
Don't know if it will help you, but ElleKayEm just figured out why it didn't work for me.  I'm on a Mac, using Safari as my browser.  Safari was opening the zip file as it downloaded, so I wasn't able to load it into Calibre.  I had my husband download it on his computer (Microsoft Edge) and then give the file to me, and it worked perfectly.

**Comment by ElleKayEm** [comment]
If you are on calibre 5.x, download the beta DeDRM plugin 7.0.3 from here:

https://github.com/apprenticeharper/DeDRM_tools/releases/tag/7.0.3

Do NOT unzip DeDRM_plugin_7.0.3.zip.  Load the file as is.


If you are on calibre 4.23 or earlier, download the DeDRM tools archive 6.8.1 from here:

https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v6.8.1

Unzip DeDRM_tools_6.8.1.zip.  Load DeDRM_Plugin.zip.

**Reply by sladetlc** [reply]
Hello, 
I am having the same issue and could use a little help. I followed the above suggestion but when I download the "assets" linked on the 7.0.3 page I do not get a file titled "DeDRM_plugin_7.0.3.zip". I only get "DeDRM_tools-7.0.3.tar", "DeDRM_tools-7.0.3", and "DeDRM_plugin_7"

any ideas?

**Reply by sladetlc** [reply]
NVM somehow it just worked!!
